src/cloud-api-adaptor/podvm-mkosi/Makefile

```diff
@@ -71,10 +79,10 @@ image: insecure-builder
 	rm -rf ./build
 	@echo "Building image..."
 ifeq ($(SE_BOOT),true)
-	sudo mkosi --profile production --image system
+	$(call run_mkosi_in_container,--profile production --image system)
```

Comment:
Sorry, I know this wasn't your change, but I think the SE_BOOT prod version should have `--tools-tree=default --tools-tree-release=40` too, so if you could fix that, it would be great.

Reply:
yes, sir!

Reply:
Thank you!